src/button/template/componentTemplate.jsx

```diff
@@ -306,6 +321,9 @@ function renderButton({ size, label, color, locale, branding, multiple, layout,
             tabindex={ hasTabIndex && 0 }>
             
             { contentText }
+            {
+                impression  && <img class='tracking-beacon' src={ impression } />
```

Comment:
Can we use the same beacon component? otherwise this image will be displayed as a broken link

Reply:
ok